configure

```diff
@@ -6337,6 +6369,12 @@ if test "$_opengl_mode" != none ; then
 					;;
 			esac
 			;;
+		sailfish)
+		        _opengl_mode=gles2
+		        _opengl_glad=yes
+			OPENGL_CFLAGS=`$_pkgconfig --cflags glesv2 egl`
+			OPENGL_LIBS=`$_pkgconfig --libs glesv2 egl`
```

Comment:
I would have expected this to not be needed since we're using GLAD. Does it work without these two lines?

Reply:
Removed